src/buyingcatalogue/values.yaml

```diff
@@ -362,6 +362,18 @@ ordapi:
       baseUriConfig:
         name: "bc-buyingcatalogue"
         key: issuer-url
+    email:
+      hostUriConfig:
+        name: "bc-buyingcatalogue"
+        key: email-url
+      port: 587
+      authenticationRequired: false
```

Comment:
launch-or-update-azure.sh will also need updating.
TBH, it might be worth taking the opportunity to refactor how email is managed in the charts, as it's a bit messy.

Reply:
Why does launch-or-update-azure.sh need updating? I might will need guidance on this one am afraid.

Reply:
So got a bit more feedback from @stuart-bass-bjss. There is a launch/update script that allows the releases to pass custom values / secrets for an environment. As it stands this includes some ISAPI configuration for email settings. This also needs to be applied for the ORDAPI configuration.

Really this needs to be configurable in a central location rather than having specific application configuration in a launch script.